applications/nrf_desktop/src/modules/dfu_mcumgr.c

```diff
@@ -110,6 +117,47 @@ static struct mgmt_callback cmd_recv_cb = {
 	.event_id = MGMT_EVT_OP_CMD_RECV,
 };
 
+#if CONFIG_DESKTOP_DFU_BACKEND_MCUBOOT
+static void dfu_backend_init(void)
+{
+	if (!IS_ENABLED(CONFIG_DESKTOP_DFU_MCUMGR_MCUBOOT_DIRECT_XIP)) {
+		int err = boot_write_img_confirmed();
+
+		if (err) {
+			LOG_ERR("Cannot confirm a running image: %d", err);
+		}
+	}
+
+	LOG_INF("MCUboot image version: %s", CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION);
+}
+#elif CONFIG_DESKTOP_DFU_BACKEND_SUIT
+static void dfu_backend_init(void)
+{
+	/* Manifest class ID
+	 * nRF Desktop uses default SUIT manifest file (default namespace and name).
+	 */
+	const suit_manifest_class_id_t manifest_class_id = {{
+		/* RFC4122 uuid5(uuid5(uuid.NAMESPACE_DNS, 'nordicsemi.com'),
+		 *              'nRF54H20_sample_root')
+		 */
+		0x3f, 0x6a, 0x3a, 0x4d, 0xcd, 0xfa, 0x58, 0xc5,
+		0xac, 0xce, 0xf9, 0xf5, 0x84, 0xc4, 0x11, 0x24
+	}};
```

Comment:
I wonder if we could not simplify it by doing the https://nordicsemi.atlassian.net/browse/NCSDK-25024 already here.
It seems it would be just one API call to add here and we already had to add this logic here.
However, I'll won't insist it now as it is in separate ticket.

Reply:
Simplifying implementation in scope of this PR may actually be a good idea (change should be really small and easy to apply)